gammapy/spectrum/flux_point.py

```diff
@@ -1082,6 +1082,10 @@ def __init__(self, model, data, mask=None, likelihood="chi2"):
                 " either 'chi2' or 'chi2assym'"
             )
 
+    def data_shape(self):
+        """Shape of the flux points data"""
+        return self.data.e_ref.shape
```

Comment:
Same comment here. Should it be the shape or `e_ref` itself that is equal?

Reply:
Yes, `e_ref` is the reference energy for the data...